.circleci/config.yml

```diff
@@ -69,8 +69,13 @@ do_steps: &do_steps
       name: run tests
       command: |
         mkdir -p ~/junit
-        make -C tools/lkl run-tests tests="--junit-dir ~/junit"
+        if [[ $VALGRIND != 1 ]]; then
+          make -C tools/lkl run-tests tests="--junit-dir ~/junit"
+        else
+          make -C tools/lkl run-tests tests="--junit-dir ~/junit boot.sh"
```

Comment:
Should we not run valgrind on other lkl binaries like net-test or disk? Or is that taking too long?

Reply:
Indeed it takes long (https://circleci.com/gh/libos-nuse/lkl-linux/1411, more than 1hr) but I changed this because
- we previously tested only with boot.sh (https://github.com/lkl/linux/commit/eefd5fba02a6baa85d9fac403cf442776603be8e#diff-5aed2ab33437efe3b515c8ab0518ac70L3)
- I saw a test failure with whole test (https://circleci.com/gh/libos-nuse/lkl-linux/1411): hijack pipe test was failed

but the pipe test also failed with normal test (https://circleci.com/gh/libos-nuse/lkl-linux/1447) so I'm going to investigate it.

Reply:
fixed: reverted the previous one.